### PR TITLE
Fix #41

### LINKED ISSUE #41: Training notebook `tpv13` does not run inside docker container
Training notebook `tpv13` does not run inside docker container. Specifically this is the visualisation with Paraview part.

Here the command is
~~~
docker cp my_docker:/home/training/tpv13/my_output .
~~~

Adding the `!` needed for calling the bash script:
~~~
docker cp my_docker:/home/training/tpv13/my_output .
~~~
gives output:
~~~
/bin/bash: line 1: docker: command not found
~~~

Maybe the docker container does not include docker?

### PR DESCRIPTION
**Pull request by sebwolf-de**

### CODE REVIEW

File: tpv13/tpv13.ipynb [hunk 1]
```diff
@@ -282,7 +282,7 @@
    "source": [
     "You can also visualize the SeisSol output in Paraview on your local machine. For that, you have to copy the files from your Docker container. \n",
     "\n",
-    "Do you remember the name of your container? Type the following command in the terminal of your local machine and replace my_docker by the name of your container and my_output by the output file you want to copy: \n",
+    "The following commands allow you to copy the simulation output from docker to the native filesystem of your maschine. Type the commands in the terminal of your local machine: \n",
```
Comment (Thomas-Ulrich) [inline tpv13/tpv13.ipynb:285]: typo maschine

File: tpv13/tpv13.ipynb [hunk 2]
```diff
@@ -292,7 +292,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "docker cp my_docker:/home/training/tpv13/my_output ."
+    "# Execute on your local machine, not within this notebok!",
```
Comment (Thomas-Ulrich) [inline tpv13/tpv13.ipynb:295]: typo notebook

File: tpv13/tpv13.ipynb [hunk 3]
```diff
@@ -292,7 +292,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "docker cp my_docker:/home/training/tpv13/my_output ."
+    "# Execute on your local machine, not within this notebok!", 
+    "\n",
+    "export my_docker=!(docker ps --format \"{{.Names}}\")",
```
Comment (Thomas-Ulrich) [inline tpv13/tpv13.ipynb:297]: remove !
?

**Review by Thomas-Ulrich — APPROVED** [review]

File: tpv13/tpv13.ipynb (hunk 3 above)
Comment (sebwolf-de) [reply tpv13/tpv13.ipynb]: Has to be a `$` to store output in a bash variable, thanks for noticing!

**Review by Thomas-Ulrich — APPROVED** [review]

**Pull request merged**